src/main/java/org/sefglobal/scholarx/service/ProgramService.java

```diff
@@ -171,6 +213,26 @@ public void deleteProgram(long id) throws ResourceNotFoundException {
         }
     }
 
+    /**
+     * Retrieves all the {@link Mentee} objects filtered from {@link Program} {@code id}
+     *
+     * @param id     which is the Program id of the filtering {@link Mentor} objects
+     * @return {@link List} of {@link Mentee} objects
+     *
+     * @throws ResourceNotFoundException if the requesting {@link Program} to filter
+     *                                  {@link Mentee} objects doesn't exist
+     */
+    public List<Mentee> getAllMenteesByProgramId(long id)
```

Comment:
Can you create the unit test for this method?

Reply:
Let's complete the testing on a later stage @anjula-sack create an issue for this.

Reply:
I created an issue for this.